src/main/java/cn/nukkit/level/biome/Biome.java

```diff
@@ -110,6 +51,25 @@ public static Biome getBiome(String name) {
         return null;
     }
 
+    private static int generateBiomeColor(double temperature, double rainfall) {
```

Comment:
biome color it's not by clientside?

Reply:
it is, but nukkit stores biome colors anyway (old code that wasn't removed). just ignore it, i'll remove that as part of this PR too most likely.

Reply:
ok